libs/ledger-live-common/package.json

```diff
@@ -70,6 +70,8 @@
     "test": "pnpm ci-test-unit",
     "ci-test-unit": "env-cmd -f .ci.unit.env pnpm jest --ci --updateSnapshot && git diff --exit-code src",
     "ci-test-integration": "env-cmd -f .ci.integration.env pnpm jest --ci --updateSnapshot --passWithNoTests",
+    "test-bridge": "env-cmd -f .ci.bridge.env pnpm jest --ci --updateSnapshot --passWithNoTests --runInBand",
```

Comment:
just a question, should we still need "updateSnapshot" for "test-bridge" and "test-bridge-update" as we've mocked the http responses now ?

Reply:
:thinking: Indeed we could restrict to standard integration bridge test